### PR TITLE
Keadm: use latest version in help info

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What this PR does / why we need it**:

The help info should better use latest release version.

Do not call `curl` to get since it should be updated in release process.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: pkg/version/version.go [hunk 1]
```diff
@@ -40,3 +40,7 @@ func Get() apimachineryversion.Info {
 		Platform:     fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+func GetLatestVersion() string {
+	return "1.4.0"
```
Comment (kevin-wangzefeng) [inline pkg/version/version.go:45]: You can get from https://kubeedge.io/latestversion

Comment (daixiang0) [reply pkg/version/version.go:45]: This can work with offline case, and update all version info in release process is a common way.

Comment (fisherxu) [reply pkg/version/version.go:45]: We have a default version in `keadm/cmd/keadm/app/cmd/common/constant.go`

Comment (daixiang0) [reply pkg/version/version.go:45]: Let's use it :)

**Pull request merged**